packages/api/controllers/program.js

```diff
@@ -0,0 +1,51 @@
+import shortId from 'shortid'
```

Comment:
Nice callout! Looks like this happened 7 months ago. Actually good that you called this out anyway since you mentioned you want to use full on guids for IDs. Do you want me to swap this out for a guid generator? I much prefer these shorter IDs if I can change your mind ;) Take a read of the nanoId docs and let me know.

Reply:
Also, see https://zelark.github.io/nano-id-cc/. I'd like to change the size down to 10 which comes out as "~2 thousand years needed, in order to have a 1% probability of at least one collision." if we had 10 ids generated every hour.

Reply:
It's not a big deal. Just go with nano-id and call it good for now :) We can worry about cryptographic security later